mods/tags.py

```diff
@@ -132,9 +132,10 @@ def prepare_message_hook(self, request, message, detailed):
         ob = message.get_property("obsoleted-by")
         if ob:
             new = Message.objects.find_series(ob, message.project.name)
-            message.status_tags.append({
-                "title": "Has a newer version: " + new.subject,
-                "type": "default",
-                "char": "O",
-                })
+            if not new is None:
```

Comment:
`if new is not None` please. PEP8:

> 
> Use is not operator rather than not ... is. While both expressions are functionally identical, the former is more readable and preferred.
> 
> Yes:
> 
> if foo is not None:
> No:
> 
> if not foo is None: